### DISCUSSION TITLE
New sort by feature, key count in mania

### DISCUSSION THREAD

**Discussion by Redd56** [ppy/osu#20719] [Ideas] [answered]
Simple as that, a new sort function key count, since sometimes i want to play 5k maps, sometimes 4k, sometimes 10k but unless the map creator added 4k/5k/xk into the title of the map it doesnt show up when searching. 

This would be a mania exclusive thing since neither taiko standard or catch have use for something like that, but could be used by custom rulesets for special searches

**Comment by peppy** [comment, marked as answer]
You can do this for now using `keys:4` in the search box

**Reply by Redd56** [reply]
oh thanks did not know that, 
have a great night/day peppy
